doc/sphinx/source/framework-integration-guides.rst

```diff
@@ -46,55 +46,31 @@ system that already has an ``ngraph_dist`` installed.
    .. code-block:: console
 
       $ sudo apt-get install python3-pip python3-venv
+      $ python3 -m venv .venv
+      (.venv)$ . bin/activate
```

Comment:
type here? you need to do . .venv/bin/activate and then the (.venv) prompt will show up from the next steps.

Reply:
Rest of it LGTM.

Reply:
I prefer giving it a visible name, like `pydoc` since macos makes directories starting with `.` invisible to file finders, like PyCharm, which means you can't use them as an environment within the IDE.

Reply:
What makes the most sense is to create a venv named `frameworks`, I think.  This is kind of the line I was going down originally with my previous instructions for system install.  People will be able to have a single `venv` that represents nGraph-enabled frameworks.  Tested it; it works.  Awesome.